code/Base/Hud/Chat/ChatBox.scss

```diff
@@ -0,0 +1,150 @@
+﻿SportsChatBox {
+	position: absolute;
+	bottom: 200px;
+	left: 100px;
+	z-index: 100;
+	flex-direction: column;
+	width: 500px;
+	font-family: Poppins;
+	font-size: 13px;
+
+	.party_chat {
```

Comment:
Inconsistent naming of properties here. Should be using hyphens instead of underscores.